### DISCUSSION TITLE
Webhooks for mentions

### DISCUSSION THREAD

**Discussion by stereo9** [outline/outline#5259] [Ideas]
I am using webhooks now to check if someone from my team was mentioned in a comment and notify them via rocketchat. this works perfectly for now.

I could theoretically do the same for pages, but I feel the amount of data to be handled is simply not worth it.
Therefore it could be nice to have webhooks for mentions directly, data body including the user who mentioned, the user who was mentioned, the pageurl and title.

**Comment by tommoor** [comment]
It's a reasonable request 👍
